datafusion/core/src/datasource/physical_plan/mod.rs

```diff
@@ -23,17 +23,20 @@ mod csv;
 mod file_scan_config;
 mod file_stream;
 mod json;
+#[cfg(feature = "parquet")]
 pub mod parquet;
 
 pub(crate) use self::csv::plan_to_csv;
 pub use self::csv::{CsvConfig, CsvExec, CsvOpener};
-pub(crate) use self::file_scan_config::PartitionColumnProjector;
 pub(crate) use self::json::plan_to_json;
+#[cfg(feature = "parquet")]
```

Comment:
We can probably simplify the code a little as a follow on PR by removing this `pub(crate) use` and using it directly in the parquet module